.github/workflows/rust.yml

```diff
@@ -219,7 +219,7 @@ jobs:
   light_client_tests:
     name: "Test (Light Client)"
     runs-on: ubuntu-latest-16-cores
```

Comment:
I wonder if we really need such beefy machine for these tests ^^

Reply:
Good point, let me play around with these settings 👍

Reply:
I think it's probably compiling the tests that are slow when I tried it locally.